core/src/main/java/org/elasticsearch/common/cache/Cache.java

```diff
@@ -195,31 +196,32 @@ protected long now() {
         /**
          * get an entry from the segment
          *
-         * @param key the key of the entry to get from the cache
-         * @param now the access time of this entry
+         * @param key       the key of the entry to get from the cache
+         * @param now       the access time of this entry
+         * @param isExpired test if the entry is expired
```

Comment:
Maybe add something about how if the entry is expired, it is not removed from the cache but is instead ignored?